addons/udes_stock_refactoring/models/stock_location.py

```diff
@@ -0,0 +1,40 @@
+"""UDES Refactoring extensions for the StockLocation model."""
+
+from odoo import fields, models
+
+
+class StockLocation(models.Model):
+    """Stock Location model."""
+
+    _inherit = "stock.location"
+
+    u_is_picking_zone = fields.Boolean(
```

Comment:
This could go to `udes_stock` module, since could be a useful field for other modules.

Reply:
Will do (both fields will need to move, as refactoring depends on stock).

Reply:
Addressed in ff893c2bf7.
(I'll split out the stock and refactoring changes if we are happy with this commit).